### PR TITLE
fix sort on admin user

### PR DESCRIPTION
**Pull request by petersg83**

### CODE REVIEW

File: packages/core/admin/server/services/user.js [hunk 1]
```diff
@@ -125,7 +126,7 @@ const resetPasswordByEmail = async (email, password) => {
  * @param {int|string} userId user's id to look for
  */
 const isLastSuperAdminUser = async userId => {
-  const user = await findOne({ id: userId }, ['roles']);
+  const user = await findOne(userId, ['roles']);
```
Comment (alexandrebodin) [inline packages/core/admin/server/services/user.js:129]: if the defaultpopulate is roles you could remove it from the params

**Review by alexandrebodin — APPROVED** [review]
LGTM. One comment if you want

File: packages/core/admin/server/services/user.js (hunk 1 above)
Comment (petersg83) [reply packages/core/admin/server/services/user.js:129]: I saw that too, let's do it

**Review by alexandrebodin — APPROVED** [review]
LGTM. waiting for the tests

**Pull request merged**